src/Chapter2.hs

```diff
@@ -741,7 +759,8 @@ the list with only those lists that contain a passed element.
 
 🕯 HINT: Use the 'elem' function to check whether an element belongs to a list
 -}
-contains = error "contains: Not implemented!"
+contains :: Int -> [[Int]] -> [[Int]] 
+contains x ls = filter (elem x) ls
```

Comment:
Now, after you've mastered eta-reduction, you can apply such a technique to this function as well  🙂 
But your implementation is already great 💯

Reply:
Done, thanks!